### ISSUE TITLE
EvalError: call to eval() blocked by CSP

### ISSUE THREAD

**Issue by opalko** [freeCodeCamp/freeCodeCamp#38050]
Cannot progress past Javascript section: Basic Algorithm Scripting: Return Largest Numbers in Arrays, error in console: "EvalError: call to eval() blocked by CSP"  Running Firefox 72.0.1 on Macbook Pro High Sierra
![Screen Shot 2020-01-10 at 4 07 07 PM](https://user-images.githubusercontent.com/25775780/72189883-a64e2580-33c3-11ea-92d6-dcc7ec37d1c8.png)

**Comment by brandon-thenaro** [comment]
Duplicate of [#38044](https://github.com/freeCodeCamp/freeCodeCamp/issues/38044)

**Comment by ojeytonwilliams** [comment]
Thanks for reporting this, @opalko.  @brandon-thenaro is right, though, so I'm closing this so we can track the issue in one place.
